### DISCUSSION TITLE
Provide UTM file locally

### DISCUSSION THREAD

**Discussion by areeckmann** [naveenrajm7/vagrant_utm#3] [Q&A] [answered]
Hi, really enjoying the Plugin! Great work!

I was wondering if there is an option (that I'm missing or that is in the works) to select a utm file from local storage as opposed to downloading it from the github. 
When using `u.utm_file_url` I would love to just select a local path to the utm file.
Downloading one Gigabyte for each `vagrant destroy && vagrant up` seems a bit unnecessary.  

Kind regards!

**Comment by naveenrajm7** [comment, marked as answer]
Hello,

Thank you for trying out the plugin. I'm glad you are enjoying it!

Yes, I agree that the current method of just supporting URLs is not ideal and is a waste of bandwidth if the same images are needed. 
I initially added the support for using local utm files, but when you open local files, UTM will just create a shortcut to the VM.
When you destroy a VM you expect all VM files to be gone, but with shortcuts, deleting VMs will only delete the shortcut.
Hence, I removed support for opening local files.

That being said, I am working to add import support for UTM (https://github.com/utmapp/UTM/issues/6612). With import in place, we can load from local files and even start using the vagrant box and vagrant cloud. 

Created an issue (#4) to track this request.
